modules/bitgo/src/config.ts

```diff
@@ -116,6 +116,7 @@ export const ethGasConfigs = {
   minimumGasLimit: 30000,    // minimum gas limit a user can set for a send
   maximumGasLimit: 20000000, // Customers cannot set gas limits beyond this amount
 };
+
```

Comment:
nit: Please avoid whitespace-only changes to files unless they're generated by `prettier`.

Reply:
There were some changes earlier in this file but then I removed them after refactoring, missed this space thing.

Reply:
Done